src/Features/Core/Portable/PullMemberUp/Dialog/IPullMemberUpOptionsService.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.  
+
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.PullMemberUp;
+
+namespace Microsoft.CodeAnalysis.CodeRefactorings.PullMemberUp.Dialog
+{
+    internal interface IPullMemberUpOptionsService : IWorkspaceService
+    {
+        PullMembersUpAnalysisResult GetPullMemberUpOptions(Document document, ISymbol selectedNodeSymbol);
```

Comment:
the name of this method doesn't match it's return value.  you say "get PullMemberUpOptions" but you're not returning any sort of options.  You instead return some sort of anlaysis result.  Either the method name is off, or hte return value is off.